packages/strapi-helper-plugin/lib/src/utils/injectHooks.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+import invariant from 'invariant';
+
+export default ({ key, pluginId }) => WrappedComponent => {
```

Comment:
This file should be tested but in another PR once the priority has been developed